.github/custom-issue-template/frontend-kit/2. Setup Storybook.md

```diff
@@ -1,6 +1,11 @@
 # Background
-Storybook is a great way to build, validate, and test components. We want to ship a working storybook instance with each kit.
+
+Storybook is a great way to build and test components in a sandbox. Most teams opt out of Storybook because it takes time to setup. We should make it easy to add stories and colocate them with their components.
```

Comment:
You want to add a timebox and failsafe in this story. We found that in some cases (new framework) this may not work out of the box and it is time consuming.